src/nvim/edit.c

```diff
@@ -4114,11 +4115,10 @@ int ins_compl_add_tv(typval_T *const tv, const Direction dir, bool fast)
 }
 
 /// Returns TRUE when using a user-defined function for thesaurus completion.
```

Comment:
`TRUE` -> `true`

Reply:
When I do `src/clint.py src/nvim/edit.c` I get 294 errors, should I fix all of those? I don't think it has any relation to this vim patch.

Yes, this `TRUE` is changed.

Reply:
> should I fix all of those?

I think you just need to pass lint CI.

Reply:
That I have been passing for some time already.